src/ini.rs

```diff
@@ -193,6 +204,7 @@ impl Default for IniDefault {
             .cloned()
             .collect(),
             case_sensitive: false,
+            enable_inline_comments: true, // compatibility w/ previous versions
```

Comment:
nit: `retain compatibility with previous versions`

Reply:
Ok, was trying to keep it under 80 chars.  Will expand.